recipes/xtl/build.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+cmake -DBUILD_TESTS=OFF -DCMAKE_INSTALL_PREFIX=$PREFIX $SRC_DIR -DCMAKE_INSTALL_LIBDIR=lib
+make install
```

Comment:
Is there a make check we can run here?

Reply:
We cannot test c++14 libraries on conda-forge linux. All the QuantStack C++ libraries have a lot of CI (xtensor, xtensor-r, xtensor-julia, xtensor-python, xtensor-blas, xeus, xsimd)...

Quick note, `check` is not  a standard target. In the case of most of our packages, the testing is done with `make xtest` :)